test/config/security/policy.yaml

```diff
@@ -46,4 +46,6 @@ spec:
       - excludedPaths:
         - prefix: /metrics
         - prefix: /_internal/knative/activator/probe
```

Comment:
Is there value in having different paths?

Reply:
I actually have no idea, and was wondering too if everything will work with a common path. @tcnghia @vagababov ?

Reply:
we don't check paths. But if we're going generic, then we should pick a generic name :)

Reply:
Having a single path probably makes more sense because it makes writing policies easier and it is more future proof (when we add or remove components).

Reply:
Makes sense. Should the common path be something that we expect customer services won't have in their their application logic? Or can it be very short like "\probe". I had assumed that we had a complicated name with "\_knative\internal\..." to reduce chances of collision with customer service paths

Reply:
@yolocs do you have a recommendation here?

May be something like /healthz would work.  Also, if we allow this to be customized then our customers can avoid a collision even for shorter paths.

Reply:
Given we intercept the request it is of less importance if it matches (still a problem if the network rule matches).
But having something like `/__probe__` should mostly work.

Reply:
I would prefer `/healthz` as it seems like pretty common path for health check. My thinking is for something default, it's better to meet most common path. If a user runs into a conflict, the config is there for them to override.

Reply:
As @vagababov said, the path can be anything, so let's use something simple and if possible that is somewhat "standard" in the Kubernetes world. `/healthz` seems appropriate.